File: frontend/src/types/governance.ts
export type RiskLevel = "Low" | "Medium" | "High" | "Critical"

export interface GovernancePolicy {
  id: string
  name: string
  country: string
  category: string
  enabled: boolean
  approvalRequired: boolean
  humanReview: boolean
  riskLevel: RiskLevel
  description: string
  lastUpdated: string
}

File: backend/app/schemas/governance.py
from typing import Literal
from pydantic import BaseModel

RiskLevel = Literal["Low", "Medium", "High", "Critical"]


class GovernancePolicy(BaseModel):
    id: str
    name: str
    country: str
    category: str
    enabled: bool
    approvalRequired: bool
    humanReview: bool
    riskLevel: RiskLevel
    description: str
    lastUpdated: str
